templates/home.html

```diff
@@ -0,0 +1,48 @@
+{% extends "page.html" %}
+
+{% block main %}
+
+<div class="container">
```

Comment:
This dom tag should be merged with the main one below(the one with class `.container`). I mean ONLY one `.container` is enough.